variables.tf

```diff
@@ -243,6 +243,15 @@ variable "replication_configuration" {
         replica_modifications     = optional(bool, false)
         sse_kms_encrypted_objects = optional(bool, false)
       }))
+      metrics = optional(object({
```

Comment:
please sort this alphabetically with a newline between every multine block. 

metrics

replication_time

source_selection_criteria